### DISCUSSION TITLE
Bulk move cables

### DISCUSSION THREAD

**Discussion by dreng** [netbox-community/netbox#6166] [Q&A] [answered]
Hi,

I'm about to replace some switches which sums up to about 200 interfaces. Do I have to unlink and reconnect every single cable to the new switches or does netbox provide a feature like "take these 48 cables from this old switch and plug it 1:1 to a new device"?

**Comment by candlerb** [comment, marked as answer]
Unfortunately, you need to delete and recreate all the cables.  You cannot even move one endpoint of an existing cable.  If you're recording label/color/length then all those attributes need to be re-entered.

If this is a like-for-like, drop-in replacement of a switch (same device type) then it's much easier just to change the attributes of the device - i.e. serial number and asset tag, and MAC addresses of interfaces if they are recorded.

**Reply by dreng** [reply]
Thank you for clarification. I considered to change the attributes of the current devices but I ended up in the decision to keep them in stock as a backup. Hence, that's not a good option unfortunately.

**Reply by candlerb** [reply]
It *is* still possible:
* Apply the new asset tag and serial number to the existing device in-place
* Create a new device in Netbox
* Apply the old asset tag and serial number to this new device

You should note that Netbox *isn't* an asset management system, nor a spares inventory system.  Netbox records the current desired configuration of your network, and isn't intended to track the history of physical assets, nor their purchase history, support contracts etc. 

Netbox records *current deployments* of assets, rather than the history of assets.  Netbox could say "today, asset X1 is deployed in rack Y position Z with cables C1, C2, C3..." etc; tomorrow, asset X2 could be fulfilling this role.

Imagine a separate asset management system running alongside Netbox (which you may or may not actually have).  Consider that "serial number" and/or "asset tag" are foreign keys into this asset management system.  It then makes more logical sense when replacing a "device" in Netbox, to update the existing device to point to a new asset.
